app/components/blacklight/response/view_type_button_component.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= link_to url, title: label, class: "#{Array(@classes).join(' ')} view-type-#{ @key.to_s.parameterize } #{"active" if selected?}" do %>
   <%= icon %>
-  <span class="caption"><%= label %></span>
+  <span class="visually-hidden"><%= label %></span>
```

Comment:
does it make sense to leave `caption` there too for anyone that might have or want to target it for custom style override?

Just asking the question, you can say no!

Reply:
I don't see any harm in doing that. I've pushed this back in.